manifests/Microsoft/OneDrive/20.052.0311.0011.yaml

```
@@ -0,0 +1,14 @@
+Id: Microsoft.OneDrive
+Version: 20.052.0311.0011
+Name: OneDrive
+Publisher: Microsoft
+License: Copyright (c) Microsoft Corporation
+AppMoniker: OneDrive
+Tags: cloud,online,storage,pictures,files
+Description: Microsoft OneDrive, or simply OneDrive, is a file hosting service and synchronization service operated by Microsoft as part of its web version of Office.
+Homepage: https://onedrive.com
+Installers:
+  - Arch: x86
+    Url: https://go.microsoft.com/fwlink/p/?LinkID=2121808
+    Sha256: 560DF792B70B1698D2CD1477B883521F7F643D3E249B6B107AEEF99B25DFD24C
```

According to the version above, this should actually be:
```suggestion
    Url: https://oneclient.sfx.ms/Win/Prod/20.052.0311.0011/OneDriveSetup.exe
    Sha256: C1838D0D4B5D00714A2FDD40E550ECDDFDAC61E7A5DC1A3A65D5C66C617ACB1A
```